analyzer/codechecker_analyzer/analyzers/analyzer_base.py

```diff
@@ -46,6 +46,24 @@ def config_handler(self):
     def construct_analyzer_cmd(self, result_handler):
         raise NotImplementedError("Subclasses should implement this!")
 
+    @classmethod
+    @abstractmethod
+    def analyzer_binary(cls):
+        """
+        A subclass should have a analyzer_binary method
+        to return the bin of analyzer.
+        """
+        pass
+
+    @classmethod
+    @abstractmethod
+    def analyzer_env(cls):
+        """
+        A subclass should have a analyzer_env method
+        to return the env for analyzer.
```

Comment:
This is about about as descriptive, as saying this:
* return the context of the analyzer
* return the parameters of the analyzer
* return the stuff about the analyzer
* return the details of the analyzer
* return the analyzer data

You get my point :^) Do you mean to return the _environmental variables_? Why is this important? Do analyzers run with their own environmental variables?